.github/workflows/check.yml

```diff
@@ -0,0 +1,103 @@
+name: Check
```

Comment:
Can you please add a comment here at the top stating what this is checking

Reply:
Will do 🙂